src/button/component.jsx

```diff
@@ -1028,6 +1028,13 @@ export const Button : Component<ButtonOptions> = create({
             def() : Object {
                 return { action: 'checkout' };
             }
+        },
+
+        // its unclear if this prop is actually needed for second render
```

Comment:
Best I can tell, this prop is not needed since v4 uses funding.disallowed to prevent venmo from rendering on second render. https://github.com/paypal/paypal-checkout-components/pull/1609/files#diff-91faf8fd6bd34f82cce8f38fed64acd81ca263e9de4a00d6c2727fa4ac129b89R516